pkg/metamorph/client.go

```diff
@@ -3,12 +3,12 @@ package metamorph
 import (
 	"context"
 	"errors"
+	grpcopts "github.com/bitcoin-sv/arc/internal/grpc_opts"
```

Comment:
No alias needed as the rule has been excluded

Reply:
Removed